index/templates/index/index.html

```diff
@@ -24,7 +24,7 @@
 <!DOCTYPE html>
 {% extends "index/base.html" %}
 {% load static %}
-
+{% load filters %}
```

Comment:
This file may not need for current applications. Just use for-in-loop instead of index.